### DISCUSSION TITLE
Ability to change language

### DISCUSSION THREAD

**Discussion by mlody95pl** [bucanero/pkgi-ps3#94] [Ideas]
Hello @bucanero,

Can you add ability to change language?
Polish translation is garbage for now.
I don't even know what new function does.
Please consider add that option to make life better :)

**Comment by bucanero** [comment]
if you want to disable the Polish translation, you have 2 options:
- remove the `NP00PKGI3/USRDIR/LANG/pl.po` translation file, then it will switch back to english
- edit the `config.txt` file and set `language en`, so the app is forced to English. You can also force the language option to any other language, such as `es`, `de`, `it`.

Also, you're welcome to edit the `pl.po` translation file and provide a better/improved translation for the next release:
https://github.com/bucanero/pkgi-ps3/blob/master/pkgfiles/USRDIR/LANG/pl.po
